test/karma.conf.js

```diff
@@ -28,19 +30,24 @@ module.exports = function(config) {
       "bower_components/uri.js/src/URITemplate.js",
       "bower_components/uri.js/src/jquery.URI.js",
       "bower_components/uri.js/src/URI.fragmentURI.js",
-      "bower_components/hawtio-core/dist/hawtio-core.js",
-      "bower_components/kubernetes-container-terminal/dist/container-terminal.js",
-      "bower_components/hawtio-extension-service/dist/hawtio-extension-service.js",
-      // load up the fixtures first
+
+      // load fixtures before source
```

Comment:
Did we mean to drop `kubernetes-container-terminal/dist/container-terminal.js`?

Reply:
Yes, I ran this by Sam. It is not used in this repo. I think it was a copy/paste error at some point.

Reply:
OK great, good to drop it then.